robusta_krr/formatters/table.py

```diff
@@ -81,19 +81,16 @@ def table(result: Result) -> Table:
     table.add_column("Number", justify="right", no_wrap=True)
     if cluster_count > 1:
         table.add_column("Cluster", style="cyan")
-    table.add_column("Namespace", style="cyan")
-    table.add_column("Name", style="cyan")
-    table.add_column("Pods", style="cyan")
-    table.add_column("Old Pods", style="cyan")
-    table.add_column("Type", style="cyan")
-    table.add_column("Container", style="cyan")
+    table.add_column("Name", style="cyan", no_wrap=True)
+    table.add_column("Pods", style="cyan", no_wrap=True)
+    table.add_column("Container", style="cyan", no_wrap=True)
     for resource in ResourceType:
         table.add_column(f"{resource.name} Diff")
         table.add_column(f"{resource.name} Requests")
         table.add_column(f"{resource.name} Limits")
 
```

Comment:
you would want to add a sort here before the grouping like this for example and have the grouping as it was 
`result.scans.sort( key=lambda x: (x.object.current_pods_count, x.object.cluster, x.object.namespace, x.object.name), reverse=True)`

Reply:
I kept 
x.object.cluster, x.object.namespace, x.object.name
so the grouping stays the same

Reply:
Used the `result.scans.sort` and it works! thank you